### DISCUSSION TITLE
how to use proxy

### DISCUSSION THREAD

**Discussion by EnelKyss** [dyc3/steamguard-cli#267] [Q&A]
I see you added a proxy support. How do I use it. I have my own proxies and would like to use them :)

**Comment by dyc3** [comment]
See `steamguard --help`. It's currently not in the latest release (`v0.9.7`), so it'll be in the next release.

**Reply by EnelKyss** [reply]
Thanks for reply. Any idea when you make it ?

**Reply by dyc3** [reply]
I don't have a set release schedule, but either today or tomorrow.
